docs/features/software-templates/experimental.md

```diff
@@ -13,7 +13,64 @@ Please leave feedback on these features in the [Backstage Discord](https://disco
 
 ## Retries and Recovery
 
-### TODO
+Do you have long-running tasks, and can they recover and proceed after Backstage redeploy?
+
+Then you definitely will benefit from enabling the experimental scaffolder task recovery feature.
+
+Whenever you do redeploy, on startup there will be a check of all tasks in "processing" state that you identified in your template as they are capable of starting over.
+
+This is an example of how you can do it:
```

Comment:
Looking forward to trying this! Just wondering, is it better to try and explain to the reader why this was implemented (to solve the problem of) instead of a nice feature you can enable? I think it adds more weight and context.

When I read it what I am imagining is that I may use this if I have slow/long scaffolder tasks and if someone redeploys the Backstage app it will fail/lose any and all progress (the problem) and then this would be the solution right? LMK if I am misunderstanding 😅

Reply:
Hey @Parsifal-M thank you for your feedback. That's quite valid point. 
Do you think pointing to BEP would be sufficient here? Just to not duplicated the same information here?

https://github.com/backstage/backstage/tree/master/beps/0004-scaffolder-task-idempotency

Reply:
Hey! Hmm, good point I can't think of any major issues with pointing to the BEP as reference at least, only maybe later on if things start to drift slightly 😅  but I think it should be fine

Having read your other explanation further down in another [comment](https://github.com/backstage/backstage/pull/28748#discussion_r1947013146) I think this is also probably fine as is 😃